### ISSUE TITLE
error message.

### ISSUE THREAD

**Issue by emmetiray** [swillisart/BifrostEXR#6]
Hi,
first of all thank you for BifrostEXR. I think it's exactly what i was looking for.
I was trying to write the point position data of bifrost particles into the rgb channels of a exr image file.
Would like to get that information into nuke to render it with a point renderer. i piped the xyz positions into
rgb(a) and defined "height" and "width" (1024x1024). i piped this "object" with all the data into the image input of write_exr but i get this error message:

"vnnConnect "|bifrostGraph1|bifrostGraphShape1" "/set_property2.out_object" "/Write_Exr.image";
// Error: Error: while compiling top level compound 'bifrostGraphShape1', around the port definition 'Image::Write_Exr.image':
    Port 'image' has undefined type 'Object'. // "

could you send me some kind of example screenshot or file to find out what's wrong in my setup?
that would be super nice.

thx
jakob

**Comment by swillisart** [comment]
Currently the write node isn't very robust and will only take a flat array of pixels named **rgba** as a float vector4, 
width and height properties should be integers too.
It's not a geometric type so if you are setting properties make sure they are regular and not *geo property*.

![image](https://user-images.githubusercontent.com/33715288/211869896-48dfc1ab-8968-4c69-8069-ecd5afb84dc1.png)

**Comment by emmetiray** [comment]
Hey, thanks for replying this fast. Do i need a specific bifrost version? Because i get the same error by just doing this:
![bifrost](https://user-images.githubusercontent.com/114584321/212059961-6dcb90c2-a037-48e3-b8b4-ee1c004e5a25.PNG)

and is it possible to render sequences with write_exr?

**Comment by swillisart** [comment]
Yes you can read and write sequences by plugging a Time node’s frame port into a string and formatting the output path. Which version of Bifrost are you using?

**Comment by anderson-go** [comment]
Oh dear. I am already being pulled out by the Read_Exr node. I justr tried the same structures as above and didn't get pass the error below...Also, what could be wrong with this little test? Am I loading the file correctly? Do you have a working sample.exr file I can use for testing, or any specific setup to build an exr?

**Bifrost 2.7.1 Maya 2023.2**

```
// Error: Error: while compiling top level compound 'bifrostGraphShape1', around <no specific location>:
//     JIT compilation used external symbol '?read_exr@io@@YAXAEAVObject@Bifrost@@AEBVString@Amino@@@Z' which could not be resolved
```

<img width="1223" alt="bli" src="https://github.com/swillisart/BifrostEXR/assets/50350906/1c79962b-e0e5-499b-8981-73974bb5ddb1">
